inspirehep/modules/workflows/views.py

```diff
@@ -776,14 +776,19 @@ def start_workflow_for_literature_submission():
     submission_data['acquisition_source']['submission_number'] = str(workflow_object.id)
     workflow_object.data = submission_data
     workflow_object.extra_data['formdata'] = json['form_data']
-    workflow_object.extra_data['source_data'] = {
-        'extra_data': copy.deepcopy(workflow_object.extra_data),
-        'data': copy.deepcopy(workflow_object.data),
-    }
+
     # Add submission pdf from formdata.url
     form_url = workflow_object.extra_data['formdata'].get('url')
     if form_url and 'arxiv.org' not in form_url:
         workflow_object.extra_data['submission_pdf'] = form_url
+
+    # Remember that source_data should be created at the end, with all fields already
+    # filled. As first step in Wf will overwrite exerythiong with data form source_data
```

Comment:
I'd just remove this comment :)

Reply:
The problem was exactly because no one noticed that previous fix was adding `submission_pdf` after setting source_data. I added it so this not happen again. Why you would like to remove this comment?

Reply:
if you'd like to keep it's ok just please fix the typos :)

```suggestion
    # filled. As first step in Wf will overwrite everything with data form source_data
```

Reply:
:+1: